### PR TITLE
Added advanced draft of inventory management use case

### PR DESCRIPTION
**Pull request by MikeAdamsMews**
### Summary

* Added advanced draft of inventory management use case
* Note there are a few areas awaiting additional information
* And the whole use case needs a proper review form the API Team

### Checklist

- [ ] Documentation follows the [Style Guide](https://mews.atlassian.net/wiki/x/KJAoCw)
- [ ] JSON examples updated
- [ ] Properties in JSON examples are in the same order as in property tables
- [ ] Changelog dated the day when PR merged
- [ ] Changelog accurately describes all changes
- [ ] Changelog highlights the affected endpoints or operations
- [ ] Changelog highlights any deprecations
- [ ] All hyperlinks tested
- [ ] Deprecation Table updated if any deprecations
- [ ] SUMMARY.md updated if new pages added

### CODE REVIEW

File: use-cases/inventory.md [hunk 1]
```diff
@@ -1,30 +1,257 @@
...
 
-\< Insert content here \>
+Further detail can be obtained via linked entities such as `productType` (e.g. food or beverage), `modifierSet` (product qualifiers, such as pizza toppings) and `productVariant`. These can be included in the response by using the same request but specifying the relevant include query parameters – see [Essential guide > Relationships](../essential-guide/relationships.md).
 
-\< Reference endpoints or operations />
+#### Example request:
 
-## Testing your integration
+```
+GET [PlatformAddress]/api/v2/products?include=productType,modifierSet,productVariant
```
Comment (mzishtiaq) [inline use-cases/inventory.md:28]: These should be `modifierSets`, `productVariants` respectively.

File: use-cases/inventory.md [hunk 2]
```diff
@@ -1,30 +1,257 @@
...
+* __User__ – the customer
+* __Register__ – the cash register or outlet terminal used for the transaction
+* __Items__ – an array of invoice order items
+
+These can be selectively included in the response by using the same request but specifying the relevant include query parameters, as per [Essential guide > Relationships](../essential-guide/relationships.md). Item attributes include `productName`, `quantity`, `total` and amount breakdowns for tax and discounts. Use the register `self` link to obtain the full register information, including outlet name.
+
+#### Example request:
+
+```
+GET [PlatformAddress]/api/v2/invoices?include=invoiceItem,user,register
```
Comment (mzishtiaq) [inline use-cases/inventory.md:64]: These should be `invoiceItems`. Additionally, users can included refunded invoices as well which are called `originalInvoices`.

File: use-cases/inventory.md [hunk 3]
```diff
@@ -1,30 +1,257 @@
...
+
+#### Example request:
+
+```
+GET [PlatformAddress]/api/v2/invoices?include=register
+```
+
+### 2. Fetch the register, including related outlet information
+
+Use the register identifier to obtain the register and outlet, e.g. `GET /api/v2/registers?id=eef23c03-49b9-432b-b1a3-955ea1501557,include=outlet`.
```
Comment (mzishtiaq) [inline use-cases/inventory.md:206]:
```suggestion
Use the register identifier to obtain the register and outlet, e.g. `GET /api/v2/registers/eef23c03-49b9-432b-b1a3-955ea1501557?include=outlet`.
```

File: use-cases/inventory.md [hunk 4]
```diff
@@ -1,30 +1,257 @@
...
+
+#### Example request:
+
+```
+GET [PlatformAddress]/api/v2/registers?id=eef23c03-49b9-432b-b1a3-955ea1501557,include=outlet
```
Comment (mzishtiaq) [inline use-cases/inventory.md:211]:
```suggestion
GET [PlatformAddress]/api/v2/registers/eef23c03-49b9-432b-b1a3-955ea1501557?include=outlet
```

File: use-cases/inventory.md (hunk 3 above)
Comment (mzishtiaq) [reply use-cases/inventory.md:206]: This needs to be changed because the id is path param, not a query param.

File: use-cases/inventory.md [hunk 5]
```diff
@@ -1,30 +1,257 @@
...
+* [Fetching sales data](#fetching-sales-data)
+* [Fetching the location or outlet](#fetching-the-location-or-outlet)
+* [Frequently Asked Questions](#frequently-asked-questions)
+* [Additional help](#additional-help)
 
-\< Insert content here \>
+## Product synchronization
 
-\< Reference endpoints or operations />
+In order to synchronize products with Mews POS, use the [Get products](../operations/products.md#get-products) endpoint `GET /api/v2/products`. You can call this API operation periodically to refresh the product catalogue and make sure it is in sync with your system. The product attributes in the response include name, description, [SKU](https://en.wikipedia.org/wiki/Stock_keeping_unit), barcode, and price information.
```
Comment (MirkoC) [inline use-cases/inventory.md:15]: We also have `status` field but I'm not sure if that is important.

File: use-cases/inventory.md [hunk 6]
```diff
@@ -1,30 +1,257 @@
...
-> **Editable History Window**: For a full description and links to further information, see the [Mews Glossary for Open API users](https://help.mews.com/s/article/Mews-Glossary-for-Open-API-users?language=en_US).
+#### Example request:
 
-## Periodic update
+```
+GET [PlatformAddress]/api/v2/products
+```
 
-\< Insert content here \>
+Further detail can be obtained via linked entities such as `productType` (e.g. food or beverage), `modifierSet` (product qualifiers, such as pizza toppings) and `productVariant`. These can be included in the response by using the same request but specifying the relevant include query parameters – see [Essential guide > Relationships](../essential-guide/relationships.md).
```
Comment (MirkoC) [inline use-cases/inventory.md:23]: Maybe make modifierSet and productVariant to be in plural (to match the exact relationship name).

File: use-cases/inventory.md (hunk 1 above)
Comment (MirkoC) [reply use-cases/inventory.md:28]: ☝️

File: use-cases/inventory.md [hunk 7]
```diff
@@ -1,30 +1,257 @@
...
-\< Insert content here \>
+### Pagination
 
-\< Reference other content as needed />
+The response is paginated using cursor pagination. Use the `next` link to request the next page of data.
 
-## Additional help for working with the demo environment
+## Fetching sales data
+
+Use the [Get invoices](../operations/invoices.md#get-invoices) endpoint `GET /api/v2/invoices` to fetch invoices, containing order item data. Invoice attributes in the response include a description field and information about amounts, including `tax`, `total`, `subtotal`, `discount` and `tipAmount`.
```
Comment (MirkoC) [inline use-cases/inventory.md:37]: Also `cancelled` and `cancelled reason` fields are there, but again, not sure if it is important here.

Comment (jose-izquierdo) [inline use-cases/inventory.md:37]: Also, we have `discount_amount` and `item_discount_amount` but not sure either if those are important.

**Pull request merged**